dotnet/defs.bzl

```diff
@@ -60,12 +60,12 @@ def _get_runtime_runtime_identifier(rid):
     else:
         return select(
             {
-                "@rules_dotnet//dotnet/private:linux-x64": "linux-x64",
-                "@rules_dotnet//dotnet/private:linux-arm64": "linux-arm64",
-                "@rules_dotnet//dotnet/private:macos-x64": "osx-x64",
-                "@rules_dotnet//dotnet/private:macos-arm64": "osx-arm64",
-                "@rules_dotnet//dotnet/private:windows-x64": "win-x64",
-                "@rules_dotnet//dotnet/private:windows-arm64": "win-arm64",
+                "//dotnet/private:linux-x64": "linux-x64",
```

Comment:
Here and everywhere below, repo-relative labels in macros should be wrapped in `Label(...)`. Otherwise they will be interpreted relative to the BUILD file that is ultimately using them, which is likely not in `@rules_dotnet`.